src/main/resources/application.yml

```diff
@@ -154,7 +151,37 @@ cc-management-service:
           - covidcertificate/exceptional
         start: 2020-01-01T00:00:00.000Z
         end: 2022-02-16T00:00:00.000Z
-
+  cleanup:
+    cron: "-" ## deactivate
+    spots:
+      management-db:
+        sql-query:
+          count:    "select count(*) from kpi where timestamp < ?"
+          delete-all:    "delete from kpi where timestamp < ?"
```

Comment:
Aha, a delete-all isn't really a deletion of all entries in the db. In that case the naming is a bit misleading.